xrviz/utils.py

```diff
@@ -0,0 +1,35 @@
+
+
+def convert_widget(source, target):
```

Comment:
This is interesting code that we should talk about. It should have tests.

Reply:
We discussed about this in e-meet.

Reply:
I have added tests for this.